checkov/terraform/checks/resource/azure/SGRuleSSHAccessRestricted.py

```diff
@@ -0,0 +1,40 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+from checkov.common.util.type_forcers import force_list
+import re
+
+PORT_RANGE = re.compile('\d+-\d+')
+
+
+class SGRuleSSHAccessRestricted(BaseResourceCheck):
```

Comment:
@arielkru Can we build a base `SGRulePortAccessRestricted` class, and create 2 classes that inherit from it - SSH & RDP?